### PR TITLE
redo pull request

### PR DESCRIPTION
**Pull request by MLuce**

### CODE REVIEW

**Comment by MLuce** [comment]
Redid pull request

File: test/pantry_test.rb [hunk 1]
```diff
@@ -1,7 +1,123 @@
...
+    r_1.add_ingredient("Flour", 20)
+    pantry.add_to_shopping_list(r_1)
+    r_2 = Recipe.new("Spaghetti")
+    r_2.add_ingredient("Spaghetti Noodles", 10)
+    r_2.add_ingredient("Marinara Sauce", 10)
+    r_2.add_ingredient("Cheese", 5)
+    pantry.add_to_shopping_list(r_2)
+    expected = "* Cheese: 25\n* Flour: 20\n* Spaghetti Noodles: 10\n* Marinara Sauce: 10"
+    assert_equal expected, pantry.print_shopping_list
+    pantry.print_shopping_list
```
Comment (BrianZanti) [inline test/pantry_test.rb]: This line seems unnecessary

File: test/pantry_test.rb [hunk 2]
```diff
@@ -1,7 +1,123 @@
...
+
+    r3 = Recipe.new("Peanuts")
+    r3.add_ingredient("Raw nuts", 10)
+    r3.add_ingredient("Salt", 10)
+
+    pantry.add_to_cookbook(r1)
+    pantry.add_to_cookbook(r2)
+    pantry.add_to_cookbook(r3)
+
+    assert_equal ["Cheese Pizza", "Pickles", "Peanuts"], pantry.cookbook.keys
```
Comment (BrianZanti) [inline test/pantry_test.rb]: A more specific assertions would be:

```
assert_equal [r1, r2, r3], pantry.cookbook
```

File: lib/pantry.rb [hunk 3]
```diff
@@ -1,3 +1,62 @@
...
+    @cookbook = {}
+    @ingredient_list = {}
+  end
+
+  def stock_check(grocery_item)
+    if @stock[grocery_item] == nil
+      return 0
+    else
+      @stock[grocery_item]
+    end
```
Comment (BrianZanti) [inline lib/pantry.rb]: You could avoid the conditional block by giving the hash a default value of zero

File: lib/pantry.rb [hunk 4]
```diff
@@ -1,3 +1,62 @@
...
+  def restock(grocery_item, quantity)
+    if @stock[grocery_item] != nil
+      @stock[grocery_item] += quantity
+    else
+      @stock[grocery_item] = quantity
+    end
+  end
+
+  def add_to_shopping_list(recipe)
+    recipe.ingredients.map do |ingredient, quantity|
```
Comment (BrianZanti) [inline lib/pantry.rb]: This should be each instead of map since the mapped return value is not being used

Comment (BrianZanti) [reply lib/pantry.rb]: It works, it just makes me think the map is doing something when really it isn't

**Pull request closed without merging**